weaviate/connect/v4.py

```diff
@@ -393,41 +415,29 @@ def delete(
         """
         Make a DELETE request to the Weaviate server instance.
 
-        Parameters
-        ----------
-        path : str
-            Sub-path to the Weaviate resources. Must be a valid Weaviate sub-path.
-            e.g. '/meta' or '/objects', without version.
-        weaviate_object : dict, optional
-            Object is used as payload for DELETE request. By default None.
-        params : dict, optional
-            Additional request parameters, by default None
-
-        Returns
-        -------
-        httpx.Response
-            The response, if request was successful.
-
-        Raises
-        ------
-        httpx.ConnectionError
-            If the DELETE request could not be made.
+        Arguments:
+            `path`:
+                Sub-path to the Weaviate resources. Must be a valid Weaviate sub-path.
+                e.g. '/meta' or '/objects', without version.
+            `weaviate_object`:
+                Object is used as payload for DELETE request. By default None.
+            `params`:
+                Additional request parameters, by default None
+
+        Returns:
+            `httpx.Response`
+                The response, if request was successful.
+
+        Raises:
+            `httpx.ConnectionError`
+                If the DELETE request could not be made.
```

Comment:
these docstrings are competely internal - should we just delete them? They are just the same for all of the methods anyways